images/capi/ansible/windows/roles/runtimes/tasks/containerd.yml

```
@@ -0,0 +1,62 @@
+# Copyright 2020 The Kubernetes Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Download containerd
+  win_get_url:
+    url: '{{ containerd_url }}'
+    dest: '%TEMP%\containerd.tar.gz'
+    checksum: '{{ containerd_sha256 }}'
+    checksum_algorithm: "sha256"
+  register: containerd
+
+- name: Create containerd directory structure
+  win_file:
+    path:  "{{ item }}"
+    state: directory
+  loop: 
+    - '%programfiles%\containerd'
+    - '%ALLUSERSPROFILE%\containerd\state'
+    - '%ALLUSERSPROFILE%\containerd\root'
+
+- name: Unpack containerd binaries
+  win_command: cmd /c tar -zxvf {{ containerd.dest }} -C "%programfiles%\containerd"
+
+- name: Add containerd to path
+  win_path:
+    elements:
+    - '%programfiles%\containerd'
```

Looking at [the containerd repo](https://github.com/containerd/cri/blob/c8dba73eaefb1b9e91df0cb5f23dc3b275d85827/hack/release-windows.sh) I think containerd installs to a bin directory

```suggestion
    - '%programfiles%\containerd\bin'
```

---

I added `--strip components` to remove the bin during extraction